internal/dotnet/dotnet.go

```diff
@@ -25,11 +26,24 @@ func GenerateDockerfile(meta types.PlanMeta) (string, error) {
 		RUN dotnet publish -c release -o /app
 		
 		# final stage/image
+	`
+
+	if framework == "blazorwasm" {
+		dockerfile += `
+		FROM nginx:alpine AS final
+		WORKDIR /usr/share/nginx/html
+		COPY --from=build /app/wwwroot .
+		COPY nginx.conf ./
```

Comment:
zbpack should take care of `nginx.conf` for user, they don't need to provide their `nginx.conf` in source code.

You can refer to [internal/nodejs/templates/nginx-runtime.Dockerfile](https://github.com/zeabur/zbpack/blob/10c6f5fc8b34b4f87997fbd3a54b30ffb873d49d/internal/nodejs/templates/nginx-runtime.Dockerfile) to see how Node.js provider of zbpack using the Go's template package to generate nginx.conf dynamically.

Reply:
@yuaanlin I've updated the PR, could you please review once again?